epoxy-adapter/src/main/java/com/airbnb/epoxy/ChangeAnimationDelegator.java

```diff
@@ -0,0 +1,168 @@
+package com.airbnb.epoxy;
+
+import android.animation.AnimatorListenerAdapter;
+import android.animation.EpoxyItemAnimation.AnimatorListener;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView.ItemAnimator;
+import android.support.v7.widget.RecyclerView.ViewHolder;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * ItemAnimator that wraps any other ItemAnimator and delegates change animations to
+ * {@link EpoxyModel}
+ */
+public final class ChangeAnimationDelegator extends ItemAnimatorDecorator
```

Comment:
I'm not a fan of that convention tbh. When I'm reading code and encounter a library I don't know, I just think "what the hell is an Epoxy?". It doesn't really make the code any more understandable and on the off chance that I do need to know where a class comes from, I can just middle click to go to definition.

We can add the Epoxy prefix to follow the convention (which is most important after all) but maybe we should come up with a different name then, since it's pretty long already.